docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -51,6 +51,16 @@ The following command creates a new project on Expo servers for your app and cre
 </Step>
 
 <Step label="3">
+
+## Ensure new Android Build Infrastructure is disabled
+
+Go to [**Project Settings**](https://expo.dev/accounts/[account]/projects/[project]/settings) and disable **New Android Build Infrastructure**.
```

Comment:
```suggestion
Go to [**Project settings**](https://expo.dev/accounts/[account]/projects/[project]/settings) and disable **New Android Build Infrastructure**.
```

Nit: In the dashboard, we use "Project settings", so let's use the same case here to reference it.